cove_iati/views.py

```diff
@@ -76,3 +89,19 @@ def explore_iati(request, pk):
         db_data.rendered = True
 
     return render(request, 'cove_iati/explore.html', context)
+
+
+@require_POST
+@csrf_exempt
+def api_test(request):
+    form = UploadApi(request.POST, request.FILES)
+    if form.is_valid():
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            file_path = os.path.join(tmpdirname, form.cleaned_data['name'])
+            with open(file_path, 'wb+') as destination:
+                for chunk in request.FILES['file'].chunks():
+                    destination.write(chunk)
+            result = iati_json_output(tmpdirname, file_path, form.cleaned_data['openag'])
```

Comment:
I just noticed it doesn't give the option of doing org-ids checks. Can we add it to the form and pass it to ```iati_json_otuput```?